assets/tests/components/ladder.test.tsx

```diff
@@ -6,285 +6,252 @@ import DispatchProvider from "../../src/providers/dispatchProvider"
-      viaVariant: null,
-      operatorId: "op1",
-      operatorName: "SMITH",
-      speed: 50.0,
-      bearing: 33,
-      blockId: "block-1",
-      headwaySecs: 859.1,
-      previousVehicleId: "v2",
-      scheduleAdherenceSecs: 0,
-      scheduleAdherenceString: "0.0 sec (ontime)",
-      scheduleAdherenceStatus: "on-time",
-      scheduledHeadwaySecs: 120,
-      stopStatus: {
-        status: "in_transit_to",
-        stopId: "stop",
-        stopName: "stop",
-      },
-      timepointStatus: {
-        timepointId: "t1",
-        fractionUntilTimepoint: 0.5,
-      },
-      scheduledTimepointStatus: null,
-      routeStatus: "on_route",
-    },
-    {
-      id: "downward",
-      label: "downward",
-      runId: "run-2",
-      timestamp: 0,
-      latitude: 0,
-      longitude: 0,
-      directionId: 0,
-      routeId: "route",
-      tripId: "trip",
-      headsign: null,
-      viaVariant: null,
-      operatorId: "op2",
-      operatorName: "JONES",
-      speed: 50.0,
-      bearing: 33,
-      blockId: "block-1",
-      headwaySecs: 859.1,
-      previousVehicleId: "v2",
-      scheduleAdherenceSecs: 0,
-      scheduleAdherenceString: "0.0 sec (ontime)",
-      scheduleAdherenceStatus: "on-time",
-      scheduledHeadwaySecs: 120,
-      stopStatus: {
-        status: "in_transit_to",
-        stopId: "stop",
-        stopName: "stop",
-      },
-      timepointStatus: {
-        timepointId: "t2",
-        fractionUntilTimepoint: 0.75,
-      },
-      scheduledTimepointStatus: {
-        timepointId: "t2",
-        fractionUntilTimepoint: 0.75,
-      },
-      routeStatus: "on_route",
-    },
-    {
-      id: "notimepoint",
-      label: "notimepoint",
-      runId: "run-3",
-      timestamp: 0,
-      latitude: 0,
-      longitude: 0,
-      directionId: 0,
-      routeId: "route",
-      tripId: "trip",
-      headsign: null,
-      viaVariant: null,
-      operatorId: "op3",
-      operatorName: "XI",
-      speed: 50.0,
-      bearing: 33,
-      blockId: "block-1",
-      headwaySecs: 859.1,
-      previousVehicleId: "v2",
-      scheduleAdherenceSecs: 0,
-      scheduleAdherenceString: "0.0 sec (ontime)",
-      scheduleAdherenceStatus: "on-time",
-      scheduledHeadwaySecs: 120,
-      stopStatus: {
-        status: "in_transit_to",
-        stopId: "stop",
-        stopName: "stop",
-      },
-      timepointStatus: null,
-      scheduledTimepointStatus: null,
-      routeStatus: "on_route",
-    },
-  ]
-  const ladderDirection = LadderDirection.OneToZero
+describe("ladder", () => {
+  const originalGetBBox = SVGSVGElement.prototype.getBBox
+  const originalGetElementsByClassName = document.getElementsByClassName
 
-  const tree = renderer
-    .create(
-      <Ladder
-        timepoints={timepoints}
-        vehicles={vehicles}
-        ladderDirection={ladderDirection}
-        selectedVehicleId={undefined}
-      />
-    )
-    .toJSON()
+  beforeEach(() => {
+    SVGSVGElement.prototype.getBBox = () => {
```

Comment:
Don't we get the BBox of the HTMLDivElement, not the SvgElement? (Same for other tests that use this pattern)

Reply:
That was related to implementation details you removed and is no longer relevant.